ql/src/test/org/apache/hadoop/hive/ql/udf/generic/TestGenericUDFDateFormat.java

```diff
@@ -166,7 +166,7 @@ public void testNullFmt() throws HiveException {
   public void testWrongFmt() throws HiveException {
     GenericUDFDateFormat udf = new GenericUDFDateFormat();
     ObjectInspector valueOI0 = PrimitiveObjectInspectorFactory.writableStringObjectInspector;
-    Text fmtText = new Text("B");
+    Text fmtText = new Text("C");                                                                       // 'B' is defined as a pattern in jdk17 to represent period-of-day, changing it to pattern 'C' having no definition in jdk17
```

Comment:
Interesting. You may remove extra spaces

Reply:
Sure sure

Reply:
The extra spaces came in here due to the comment i had added for explaining the change. Removing the comment from file to make it cleaner & adding it as comment here for future review reference. 😅 
Comment: ` 'B' is defined as a pattern in jdk17 to represent period-of-day, changing it to pattern 'C' having no definition in jdk17`